Radzen.Blazor/RadzenButton.razor

```diff
@@ -60,6 +74,27 @@
     [Parameter]
     public EventCallback<MouseEventArgs> Click { get; set; }
 
+    [Parameter]
+    public bool IsBusy 
+    { 
+        get => _isBusy; 
+        set
+        {
+            if (value && !_isBusy)
+            {
+                // store the original Value of disabled property
+                // when entering busy state
+                _originalDisabled = Disabled;
+            }
+            _isBusy = value;
+            // return to original disabled state, when leaving busy state
+            Disabled = value ? value : _originalDisabled;
+        }
+    }
```

Comment:
I think this can be simplified by using a private property which returns the actual disabled state:

```
public bool IsDisabled
{
    get
    {
         return Disabled || IsBusy;
    }
}
```

And then use it instead of Disabled e.g. `<button disabled=@IsDisabled />`

If this is done IsBusy can be an auto property (eliminate _isBusy) and there would be no need to (re)store Disabled (thus eliminate _originalDisabled)

Reply:
Done